tests/baselines/reference/jsdocParseDotDotDotInJSDocFunction.errors.txt

```diff
@@ -0,0 +1,18 @@
+tests/cases/conformance/jsdoc/a.js(2,13): error TS7014: Function type, which lacks return-type annotation, implicitly has an 'any' return type.
+
+
+==== tests/cases/conformance/jsdoc/a.js (1 errors) ====
+    // from bcryptjs
+    /** @param {function(...[*])} callback */
+                ~~~~~~~~~~~~~~~~
+!!! error TS7014: Function type, which lacks return-type annotation, implicitly has an 'any' return type.
+    function g(callback) {
+        callback([1], [2], [3])
+    }
+    

```

Comment:
Can you also add test when its a construct signature (to verify no implicit any error is produced) ?

Reply:
`jsdocFunctionType.ts` has a test like this. (`@param {function(new: D, number)} dref`)